README.md

```diff
@@ -18,25 +32,23 @@ You should be able to access `biomage-utils` by typing:
 
     python3 biomage
 
-As a prerequisite for running all scripts in this repo, you will need a GitHub Personal Access
-Token with full access to your account. You can generate one
-[here](https://github.com/settings/tokens). Make sure you note down this and
-supply it when required. Utilities can accept this token in two ways:
 
-* either by having it available as the environment variable `GITHUB_API_TOKEN`
-* or by passing it as an option with the `-t` flag.
+### Passing credentials
+
+Tools typically require you to have sufficient AWS and GitHub credentials to function.
 
-For example:
-    
-    GITHUB_API_TOKEN=mytoken python3 biomage stage
+AWS credentials are automatically fetched from your `~/.aws` folder, so there is nothing
+to do as long as you have set up `aws-cli` properly.
```

Comment:
The setup of `aws-cli` also ask the default region for deployment, which will be used by `biomage-utils`. If the value is not set, then the staging script will fail.